---
source: test_programs/tests/execute.rs
expression: "&format!(\"success: {:?}\\nexit_code: {}\\n----- stdout -----\\n{}\\n----- stderr -----\\n{}\",\noutput.status.success(), output.status.code().unwrap_or(!0), stdout, stderr)"
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
warning: unused trait MyTrait
  ┌─ compile_failure/dup_trait_items_4/src/main.nr:1:7
  │
1 │ trait MyTrait {
  │       ------- unused trait
  │

error: Duplicate definitions of trait associated function with name MyItem found
  ┌─ compile_failure/dup_trait_items_4/src/main.nr:2:7
  │
2 │   let MyItem: u32;
  │       ------ First trait associated function found here
3 │   fn MyItem();
  │      ------ Second trait associated function found here
  │

Aborting due to 1 previous error
